core/sodasql/cli/ingest.py

```diff
@@ -35,7 +35,7 @@ class Table:
 
 
 def map_dbt_run_result_to_test_result(
-    test_nodes: dict[str, "DbtTestNode"],
+    test_nodes: Optional[dict[str, "DbtTestNode"]],
```

Comment:
Looks like `Optional` isn't imported, can you add it?

Alternatively, you can also use shortcut syntax here `dict[str, "DbtTestNode"] | None`

Reply:
That's strange that I didn't get an error here...
I'll change it to the more modern (i.e. without `Optional` though as I think it's better).